packages/atomic/cypress/integration/pager-assertions.ts

```diff
@@ -0,0 +1,7 @@
+import {PagerSelectors} from './pager-selectors';
+
+export function assertFocusActivePage() {
```

Comment:
Why did we put this in a separate file, if it's only used in one place inside one test ?

Reply:
I've been trying to convert new code to the same pattern as facets, where assertions are in a separate file, it does look a bit ridiculous right now though since only one assertion was extracted. I'll extract some more.

Reply:
I've been trying to convert new code to use the same pattern as facets, where assertions are in a separate file. Since most of `pager.cypress.ts` uses the old pattern, it looks a bit weird.